README.md

```diff
@@ -44,6 +44,9 @@ rosdep install --from-paths src --ignore-src -r -y
 
 # Install python dependencies
 pip3 install -r requirements.txt
+
+# Install lilypond dependencies that are necessary for visualization of the detected notes 
+sudo apt-get install lilypond 
```

Comment:
Consider changing `apt-get` to `apt`

Reply:
That is actually the same as in their docs.

Reply:
You can use both. `apt` is newer, a bit shorter and prettier (progressbar etc.). It is therefore preferred by most end-users nowadays. They use the same backend, so standard package installation works the same with both of them.

Reply:
See https://itsfoss.com/apt-vs-apt-get-difference/

Reply:
This is more fyi, you don't need to change it.